canonical-web-and-design/cookie-policy
# Modal all the things

## Done
- Fixed a bug with the control's content
- Replaced the notification with a modal for the first window
- Removed all unnecessary styling

## QA
- Run `npm install`
- Run `npm run build`
- Run `python -m SimpleHTTPServer`
- Visit http://localhost:8000
- Check the first window is a modal not a floating notification
- Check the manager still works and renders controls correctly

Fixes https://github.com/canonical-web-and-design/cookie-policy/issues/100
